proto/topo.proto

```diff
@@ -96,6 +96,8 @@ message Interface {
   string peer_int_name = 5;
   // Uid is the internal link identifier used by Meshnet.
   int64 uid = 6;
+  // Name of group to which this interface belongs
+  string group = 7;
```

Comment:
i would suggest let's seperate the proto changes from the rest of this cl

Reply:
ah, this will break our ixia-c-operator CI (that tests interface grouping as well).

i'll get back to this after checking all needs to be skipped for now (in ixia-c-operator CI).

Reply:
Done. https://github.com/google/kne/pull/138/commits/07d8a947df2354d19585156bcefad355bcd69d12

Reply:
I have created a new branch which contains changes for interface groups. Will create a PR as soon as this one is merged.
https://github.com/google/kne/tree/otg-interface-group